prover/circuits/srs_store.go

```diff
@@ -24,12 +24,14 @@ import (
 
 type SRSStore struct {
 	entries map[ecc.ID][]fsEntry
+	rootDir string
 }
 
 type fsEntry struct {
 	isCanonical bool
 	size        int
 	path        string
+	org         string
```

Comment:
what does "org" stand for ??

Reply:
Not solely related to that but the structs would deserve some more doc